news/2023/2023-10-29-community-mentorship-program-public-opening-and-winter-cycle-mentor-signups.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Community Mentorship Program Public Opening and Winter Cycle Mentor Signups"
+date: 2023-10-29 10:00:00 +0000
+---
+
+The 25th cycle of the Community Mentorship program is on its way! Get ready to delve deep into the world of mapping, modding, or storyboarding.
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
+
+Up until recently, the program didn't have a good way to consistently provide engagement for beginners that weren't picked up by a mentor. We're hoping to improve on this with a few new Discord forum channels:
+
+- **Roast before Post**: A way to rotate getting feedback between those that are also willing to give it. You "Roast" someone's work and then get to "Post" your own! Simple and effective. If you find yourself struggling to find input on something you've made, this could be the place you've been looking for.
+- **Mapping Discussion**: A place to discuss virtually anything related to mapping, for those who'd rather whip up a quick Discord thread than a forum post.
+With all of this, and some other small changes, we are also finally sharing our public invite link! You can join our discord [here](https://discord.gg/EvHqwvD), try out our new features and let us know how we can further build on them.
```

Comment:
```suggestion
- **Roast before Post:** A way to rotate getting feedback between those that are also willing to give it. You "roast" someone's work and then get to "post" your own! Simple and effective. If you find yourself struggling to find input on something you've made, this could be the place you've been looking for.
- **Mapping Discussion:** A place to discuss virtually anything related to mapping, for those who'd rather whip up a quick Discord thread than a forum post.

With all of this, and some other small changes, we are also finally sharing our public invite link! You can join our Discord server [here](https://discord.gg/EvHqwvD), try out our new features and let us know how we can further build on them.
```